SPECS/qemu/qemu.spec

```diff
@@ -472,6 +505,14 @@ Summary:        qemu-pr-helper utility for %{name}
 This package provides the qemu-pr-helper utility that is required for certain
 SCSI features.
 
+%package -n qemu-virtiofsd
+Summary: QEMU virtio-fs shared file system daemon
+Provides: vhostuser-backend(fs)
```

Comment:
Is there any version we can add for this `Provides`?

Reply:
I don't know where I would find a specific version for the vhostuser component, if there is one. I can add the global qemu version if that is sufficient?